Lib/unittest/mock.py

```diff
@@ -2220,6 +2220,17 @@ def mock_add_spec(self, spec, spec_set=False):
         self._mock_add_spec(spec, spec_set)
         self._mock_set_magics()
 
+    def reset_mock(self, /, *args, return_value=False, **kwargs):
+        if (
+            return_value
```

Comment:
~However, if the undocumented intention of `return_value` was actually to be able to pass a new return value here (I wonder if anyone's doing that in the wild?) then this would probably need to become the horrible:~

```suggestion
            return_value is not False
```

I just read the code, and it's clear from the superclass implementation that the `return_value` parameter to `reset_mock` is intended only to be a boolean.

Reply:
I would prefer to keep `if return_value` instead of `if return_value is not False` which is not very regular to see in a boolean context :)